docs/sources/upgrading/_index.md

```diff
@@ -57,6 +57,8 @@ The response body has the following schema:
     e.g. if you previously used `__project_id` then you'll need to update your relabel config to use `__gcp_resource_labels_project_id`.
   - `resource_type` has been moved to `__gcp_resource_type`
 
+### Loki
+
```

Comment:
I don't believe that these 2 added lines are correct. Line 34 already has the level 3 (H3) Loki header for within the Main/Unreleased heading/section.  I think that these 2 lines should be removed. Perhaps another reviewer could also check this detail.

Reply:
Good find! These crept in when branching from the older PR. I've removed in the most recent commit. Thanks!